Generic bug installing bindings

This issue is very hard to decripe, so please be patient....

Fresh install of the officiel Rasibian Buster on a Rpi4. 
Having Google TTS, Habmin or Rest Docs installed will result in issue installing another addon will cause some others bindings to stop, and after a few seconds they will start again.
This happens having openhab cloud, netamo, expire and zwave binding installed. They will all stop during the installation of another binding, and restart again after a few seconds.

Here is an example where I install Google TTS:

```2020-01-08 21:52:20.664 [INFO ] [io.openhabcloud.internal.CloudClient] - Shutting down openHAB Cloud service connection

==> /var/log/openhab2/events.log <==

2020-01-08 21:52:21.238 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from ONLINE to UNINITIALIZED
2020-01-08 21:52:21.250 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
2020-01-08 21:52:21.255 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from ONLINE to UNINITIALIZED
2020-01-08 21:52:21.267 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
2020-01-08 21:52:21.272 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from ONLINE to UNINITIALIZED
2020-01-08 21:52:21.291 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:52:21.858 [INFO ] [b.core.service.AbstractActiveService] - Expire Refresh Service has been shut down

==> /var/log/openhab2/events.log <==

2020-01-08 21:52:21.965 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to UNINITIALIZED (BRIDGE_UNINITIALIZED)

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:52:21.989 [INFO ] [b.core.service.AbstractActiveService] - Expire Refresh Service has been started

==> /var/log/openhab2/events.log <==

2020-01-08 21:52:21.993 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to INITIALIZING
2020-01-08 21:52:22.005 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to UNINITIALIZED (BRIDGE_UNINITIALIZED)

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:52:23.294 [INFO ] [.googletts.internal.GoogleTTSService] - Using cache folder /var/lib/openhab2/cache/org.openhab.voice.googletts
2020-01-08 21:52:23.812 [WARN ] [.googletts.internal.GoogleTTSService] - Audio format OGG_OPUS is not yet supported.

==> /var/log/openhab2/events.log <==

2020-01-08 21:52:23.823 [thome.event.ExtensionEvent] - Extension 'voice-googletts' has been installed.

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:52:24.589 [ERROR] [io.openhabcloud.internal.CloudClient] - Error connecting to the openHAB Cloud instance: not authorized

==> /var/log/openhab2/events.log <==

2020-01-08 21:52:24.811 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from INITIALIZING to ONLINE
2020-01-08 21:52:24.860 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED (BRIDGE_UNINITIALIZED) to INITIALIZING
2020-01-08 21:52:24.861 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED (BRIDGE_UNINITIALIZED) to INITIALIZING
2020-01-08 21:52:24.876 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from INITIALIZING to OFFLINE: Pending parent object initialization
2020-01-08 21:52:24.878 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from INITIALIZING to OFFLINE: Pending parent object initialization
2020-01-08 21:52:24.883 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from OFFLINE: Pending parent object initialization to ONLINE
2020-01-08 21:52:31.860 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from OFFLINE: Pending parent object initialization to ONLINE`


Next one, while having Google TTS installed, I install the Astro binding:


```2020-01-08 21:54:18.263 [INFO ] [io.openhabcloud.internal.CloudClient] - Shutting down openHAB Cloud service connection

==> /var/log/openhab2/events.log <==

2020-01-08 21:54:18.356 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from ONLINE to UNINITIALIZED
2020-01-08 21:54:18.371 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
2020-01-08 21:54:18.376 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from ONLINE to UNINITIALIZED
2020-01-08 21:54:18.392 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
2020-01-08 21:54:18.396 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from ONLINE to UNINITIALIZED
2020-01-08 21:54:18.418 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
2020-01-08 21:54:18.744 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to UNINITIALIZED (BRIDGE_UNINITIALIZED)
2020-01-08 21:54:18.783 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to INITIALIZING
2020-01-08 21:54:18.797 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to UNINITIALIZED (BRIDGE_UNINITIALIZED)

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:54:18.870 [INFO ] [.googletts.internal.GoogleTTSService] - Using cache folder /var/lib/openhab2/cache/org.openhab.voice.googletts

2020-01-08 21:54:19.286 [WARN ] [.googletts.internal.GoogleTTSService] - Audio format OGG_OPUS is not yet supported.

==> /var/log/openhab2/events.log <==

2020-01-08 21:54:19.379 [thome.event.ExtensionEvent] - Extension 'binding-astro' has been installed.

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:54:19.424 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'astro:sun:local' to inbox.

==> /var/log/openhab2/events.log <==

2020-01-08 21:54:19.425 [home.event.InboxAddedEvent] - Discovery Result with UID 'astro:sun:local' has been added.

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:54:19.441 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'astro:moon:local' to inbox.

==> /var/log/openhab2/events.log <==

2020-01-08 21:54:19.442 [home.event.InboxAddedEvent] - Discovery Result with UID 'astro:moon:local' has been added.
2020-01-08 21:54:21.583 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from INITIALIZING to ONLINE
2020-01-08 21:54:21.615 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED (BRIDGE_UNINITIALIZED) to INITIALIZING
2020-01-08 21:54:21.620 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED (BRIDGE_UNINITIALIZED) to INITIALIZING
2020-01-08 21:54:21.644 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from INITIALIZING to OFFLINE: Pending parent object initialization
2020-01-08 21:54:21.650 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from INITIALIZING to OFFLINE: Pending parent object initialization
2020-01-08 21:54:21.661 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from OFFLINE: Pending parent object initialization to ONLINE

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:54:22.318 [ERROR] [io.openhabcloud.internal.CloudClient] - Error connecting to the openHAB Cloud instance: not authorized

==> /var/log/openhab2/events.log <==

2020-01-08 21:54:26.941 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from OFFLINE: Pending parent object initialization to ONLINE```




Then I uninstall Google TTS:

```2020-01-08 21:56:50.928 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from ONLINE to UNINITIALIZED
2020-01-08 21:56:50.955 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
2020-01-08 21:56:50.959 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from ONLINE to UNINITIALIZED
2020-01-08 21:56:50.975 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
2020-01-08 21:56:50.977 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from ONLINE to UNINITIALIZED
2020-01-08 21:56:51.003 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)

==> /var/log/openhab2/openhab.log <==

2020-01-08 21:56:51.115 [INFO ] [b.core.service.AbstractActiveService] - Expire Refresh Service has been shut down
2020-01-08 21:56:51.806 [INFO ] [b.core.service.AbstractActiveService] - Expire Refresh Service has been started

==> /var/log/openhab2/events.log <==

2020-01-08 21:56:51.883 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to UNINITIALIZED (BRIDGE_UNINITIALIZED)
2020-01-08 21:56:51.921 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to INITIALIZING
2020-01-08 21:56:51.936 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to UNINITIALIZED (BRIDGE_UNINITIALIZED)
2020-01-08 21:56:52.115 [thome.event.ExtensionEvent] - Extension 'voice-googletts' has been uninstalled.
2020-01-08 21:56:54.860 [hingStatusInfoChangedEvent] - 'netatmo:netatmoapi:61f15dc2' changed from INITIALIZING to ONLINE
2020-01-08 21:56:54.880 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from UNINITIALIZED (BRIDGE_UNINITIALIZED) to INITIALIZING
2020-01-08 21:56:54.890 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from UNINITIALIZED (BRIDGE_UNINITIALIZED) to INITIALIZING
2020-01-08 21:56:54.898 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from INITIALIZING to OFFLINE: Pending parent object initialization
2020-01-08 21:56:54.904 [hingStatusInfoChangedEvent] - 'netatmo:NAMain:61f15dc2:70ee502727e6' changed from OFFLINE: Pending parent object initialization to ONLINE
2020-01-08 21:56:54.911 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from INITIALIZING to OFFLINE: Pending parent object initialization
2020-01-08 21:57:00.211 [hingStatusInfoChangedEvent] - 'netatmo:NAModule1:61f15dc2:0200002741f4' changed from OFFLINE: Pending parent object initialization to ONLINE```

Then I uninstall Astro binding: (Google TTS is no longer installed)

```2020-01-08 21:58:24.470 [thome.event.ExtensionEvent] - Extension 'binding-astro' has been uninstalled.```

And reinstall Astro binding again:
```2020-01-08 21:59:04.629 [thome.event.ExtensionEvent] - Extension 'binding-astro' has been installed.```

As mentioned, I can reproduce this issue having either Google TTS, Habmin or Rest Docs. Without these, the issue seem to be gone.
The example with Astro binding is just an example. I could have chosen anything else from PaperUI and the issue beeing the same. 
On my main system Rpi 3B+ where I´d discover this issue, which recently was updated from openhab 2.5M2 to 2.5 stable release, I have Zwave runing as well. Zwave is affected by this issue as well.

--- a/features/karaf/openhab-core/src/main/feature/feature.xml
+++ b/features/karaf/openhab-core/src/main/feature/feature.xml
@@ -74,47 +74,26 @@
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.auth.oauth2client/${project.version}</bundle>
 	</feature>
 
-	<feature name="openhab-core-automation" version="${project.version}">
-		<feature>openhab-core-base</feature>
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.automation/${project.version}</bundle>
-	</feature>
-
 	<feature name="openhab-core-automation-module-script" version="${project.version}">
 		<feature>openhab-core-base</feature>
-		<feature dependency="true">openhab-core-automation</feature>
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.automation.module.script/${project.version}</bundle>
 	</feature>
 
 	<feature name="openhab-core-automation-module-script-rulesupport" version="${project.version}">
 		<feature>openhab-core-base</feature>
-		<feature dependency="true">openhab-core-automation</feature>
 		<feature dependency="true">openhab-core-automation-module-script</feature>
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.automation.module.script.rulesupport/${project.version}</bundle>
 	</feature>
 
 	<feature name="openhab-core-automation-module-media" version="${project.version}">
 		<feature>openhab-core-base</feature>
-		<feature dependency="true">openhab-core-automation</feature>
 		<feature dependency="true">openhab-core-automation-module-script</feature>
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.automation.module.media/${project.version}</bundle>
 	</feature>
 
 	<feature name="openhab-core-automation-rest" version="${project.version}">
 		<feature>openhab-core-base</feature>
-		<feature dependency="true">openhab-core-automation</feature>
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.automation.rest/${project.version}</bundle>
-	</feature>
-
-	<feature name="openhab-core-config-serial" version="${project.version}">
-		<feature>openhab-core-base</feature>
-		<feature>openhab-core-io-transport-serial</feature>
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.serial/${project.version}</bundle>
-	</feature>
-
-	<feature name="openhab-core-config-discovery-usbserial" version="${project.version}">
-		<feature>openhab-core-base</feature>
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial/${project.version}</bundle>
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial.linuxsysfs/${project.version}</bundle>
 	</feature>
 
 	<feature name="openhab-core-io-bin2json" description="Binary to JSON converter" version="${project.version}">
@@ -227,15 +206,6 @@
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.mqtt/${project.version}</bundle>
 	</feature>
 
-	<feature name="openhab-core-io-transport-serial" version="${project.version}">
-		<feature>openhab-core-base</feature>
-		<requirement>osgi.service;filter:="(objectClass=org.openhab.core.io.transport.serial.SerialPortManager)"</requirement>
-		<requirement>osgi.service;filter:="(objectClass=org.openhab.core.io.transport.serial.SerialPortProvider)"</requirement>
-		<feature dependency="true">openhab-core-io-transport-serial-rxtx</feature>
-
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial/${project.version}</bundle>
-	</feature>
-
 	<feature name="openhab-core-io-transport-serial-javacomm" version="${project.version}">
 		<feature>openhab-core-base</feature>
 
@@ -244,29 +214,6 @@
 
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial/${project.version}</bundle>
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial.javacomm/${project.version}</bundle>
-	</feature>
-
-	<feature name="openhab-core-io-transport-serial-rxtx" version="${project.version}">
-		<feature>openhab-core-base</feature>
-
-		<requirement>openhab.tp;filter:="(&amp;(feature=serial)(impl=rxtx))"</requirement>
-		<feature dependency="true">openhab.tp-serial-rxtx</feature>
-
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial/${project.version}</bundle>
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial.rxtx/${project.version}</bundle>
-	</feature>
-
-	<feature name="openhab-core-io-transport-serial-rfc2217" version="${project.version}">
-		<feature>openhab-core-base</feature>
-
-		<requirement>openhab.tp;filter:="(&amp;(feature=serial)(impl=rxtx))"</requirement>
-		<feature dependency="true">openhab-core-io-transport-serial-rxtx</feature>
-
-		<requirement>openhab.tp;filter:="(feature=commons-net)"</requirement>
-		<feature dependency="true">openhab.tp-commons-net</feature>
-
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial/${project.version}</bundle>
-		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial.rxtx.rfc2217/${project.version}</bundle>
 	</feature>
 
 	<feature name="openhab-core-io-transport-upnp" version="${project.version}">
@@ -427,7 +374,6 @@
 		<feature>openhab-core-automation-rest</feature>
 		<feature>openhab-core-automation-module-script</feature>
 		<feature>openhab-core-automation-module-media</feature>
-		<feature>openhab-core-automation</feature>
 		<feature>openhab-core-io-console-karaf</feature>
 		<feature>openhab-core-io-http-auth</feature>
 		<feature>openhab-core-io-rest-auth</feature>
@@ -499,9 +445,23 @@
 	</feature>
 
 	<feature name="openhab-transport-serial" description="Serial Transport" version="${project.version}">
-		<feature>openhab-core-io-transport-serial-rfc2217</feature>
-		<feature>openhab-core-config-serial</feature>
-		<feature>openhab-core-config-discovery-usbserial</feature>
+		<feature>openhab-core-base</feature>
+
+		<requirement>osgi.service;filter:="(objectClass=org.openhab.core.io.transport.serial.SerialPortManager)"</requirement>
+		<requirement>osgi.service;filter:="(objectClass=org.openhab.core.io.transport.serial.SerialPortProvider)"</requirement>
+
+		<requirement>openhab.tp;filter:="(feature=commons-net)"</requirement>
+		<feature dependency="true">openhab.tp-commons-net</feature>
+
+		<requirement>openhab.tp;filter:="(&amp;(feature=serial)(impl=rxtx))"</requirement>
+		<feature dependency="true">openhab.tp-serial-rxtx</feature>
+
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.serial/${project.version}</bundle>
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial/${project.version}</bundle>
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial.linuxsysfs/${project.version}</bundle>
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial/${project.version}</bundle>
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial.rxtx/${project.version}</bundle>
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.transport.serial.rxtx.rfc2217/${project.version}</bundle>
 	</feature>
 
 	<feature name="openhab-transport-upnp" description="UPnP Transport" version="${project.version}">

